lib/cocoapods/validator.rb

```diff
@@ -857,25 +857,22 @@ def xcodebuild(action, scheme, configuration)
         command += Fourflusher::SimControl.new.destination(:oldest, 'tvOS', deployment_target)
       end
 
-      output, status = _xcodebuild(command)
-
-      unless status.success?
+      begin
+        _xcodebuild(command, true)
+      rescue
         message = 'Returned an unsuccessful exit code.'
         message += ' You can use `--verbose` for more information.' unless config.verbose?
         error('xcodebuild', message)
       end
-
-      output
     end
 
     # Executes the given command in the current working directory.
     #
-    # @return [(String, Status)] The output of the given command and its
-    #         resulting status.
+    # @return [String] The output of the given command
     #
-    def _xcodebuild(command)
+    def _xcodebuild(command, raise_on_failure = false)
```

Comment:
any reason `raise_on_failure` parameter is optional and has a default value? Why not always `true`?

Reply:
`capture_command`, which `_xcodebuild` used to call, doesn't raise exceptions. It used return codes. `execute_command` requires a `raise_on_failure` argument, though, with a default of true, which may be useful for future uses of `_xcodebuild`. I could see taking the parameter out, and always passing false to `execute_command`, though. Would you like me to change that?

Reply:
@dnkoutso Is there anything you'd like me to change, or is this still ready to merge?

Reply:
?

Reply:
Leaving this in to maintain existing behavior seems reasonable to me.